CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing to `cargo-screeps` and `screeps-game-api`:
+
+1. Ask questions. We're all in on the `#rust-wasm` channel on the [screeps slack].
+2. Make PRs early, often. We'll review code thoroughly and help where possible.
+3. Issues are your guide for what needs to be done. If you think something needs doing and it isn't
+   already an issue, make it one!
+
+# Style
+
+We adhere to the following guidelines:
+
+- Code Formatting: [fmt-rfcs Rust Style Guide]
+- API building: [Rust API Guidelines]
+
+In addition, we have the following in-house guidelines:
+
+## `use` formatting
+
+`use` statements should be at the top of the file, module or function they are used within.
```

Comment:
Prior to module documentation?

Reply:
Right, thanks for catching that. Meant after.